eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteChannelsDb.scala

```diff
@@ -164,11 +167,31 @@ class SqliteChannelsDb(val sqlite: Connection) extends ChannelsDb with Logging {
 
   override def listLocalChannels(): Seq[PersistentChannelData] = withMetrics("channels/list-local-channels", DbBackends.Sqlite) {
     using(sqlite.createStatement) { statement =>
-      statement.executeQuery("SELECT data FROM local_channels WHERE is_closed=0")
+      statement.executeQuery("SELECT data FROM local_channels WHERE is_closed=0 ORDER BY created_timestamp")
         .mapCodec(channelDataCodec).toSeq
     }
   }
 
+  private val listClosedChannelsSql = "SELECT data FROM local_channels WHERE is_closed=1 ORDER BY closed_timestamp DESC"
```

Comment:
Why is this outside of the `listClosedChannels` function? This should be a `val` directly inside that function?